Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebViewGetContents.mm

```diff
@@ -562,3 +562,12 @@ - (NSAttributedString *)_contentsAsAttributedString
 
     TestWebKitAPI::Util::run(&finished);
 }
+
+TEST(WKWebView, AttributedStringIncludesUserSelectNoneContent)
+{
+    RetainPtr webView = adoptNS([[TestWKWebView alloc] initWithFrame:CGRectMake(0, 0, 400, 300)]);
+    [webView synchronouslyLoadHTMLString:@"<body><p style='-webkit-user-select: none;'>Hello</p></body>"];
```

Comment:
Hm... doesn't this mean -webkit-user-select: none content will once again always appear in the pasted content. I feel like a better fix would be to detect whether the content came from confluence by checking class names, id, etc... in the pasted content.

Reply:
We talked about this in person. The content web archive / HTML copied into pasteboard won't contain `-webkit-user-select: none` content on websites other than Confluence due to this quirk being specific to Confluence so we wouldn't end up re-introducing `-webkit-user-select: none` content elsewhere.